Resources/Prototypes/_Harmony/Entities/Mobs/Player/animals.yml

```
@@ -0,0 +1,44 @@
+﻿- type: entity
+  name: Rampaging Gorilla
+  parent: MobGorilla
+  id: MobGorillaRampaging
+  description: A gorilla that has gone wild. Run!
```

Order is incorrect, should be type > abstract > parent > id > categories > name > suffix > description > components
```suggestion
  parent: MobGorilla
  id: MobGorillaRampaging
  name: Rampaging Gorilla
  description: A gorilla that has gone wild. Run!
```